tutorials/eth-api/chat-gpt.md

```diff
@@ -0,0 +1,166 @@
+
+### Limitations
+
+- At the time of writing, ChatGPT is in a research preview state.
+- ChatGPT can sometimes hallucinate, that is, output convincing and plausible-sounding answers that are factually incorrect. In such cases, it typically will not warn you of the inaccuracy.
+- This tutorial is for demonstration purposes only and code produced by ChatGPT should not be used in production.
+- The code generated in this tutorial is not audited, reviewed, or otherwise verified. It may contain errors and is not suitable for use in production. 
+- Prompting GPT-4 with the exact inputs specified in this tutorial will likely produce different outputs - that is expected due to ChatGPT's architecture as a language model. 
+
+![Limitations](/images/tutorials/eth-api/chatgpt/chatgpt-1.png)
+
+**Please note that the contracts we'll be creating today are for educational purposes only and should not be used in a production environment**.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+For this tutorial, you'll need the following:
+
+- A free [OpenAI account to access ChatGPT](https://chat.openai.com/){target=_blank}
+- An account funded with DEV tokens to be used on the Moonbase Alpha TestNet if you'd like to deploy any resulting contracts.
+  --8<-- 'text/faucet/faucet-list-item.md'
+
+## Sign up for an OpenAI Account {: #sign-up-for-an-openai-account }
+
+You can sign up for a free account to access ChatGPT by heading to [OpenAI's website.](https://chat.openai.com/auth/login){target=_blank} You'll need to provide both an email address and a phone number. A subscription to ChatGPT Plus is not required to complete this tutorial.
+
+![Sign up for OpenAI account](/images/tutorials/eth-api/chatgpt/chatgpt-2.png)
+
+## Create an ERC-20 Token Contract {: #create-an-erc-20-token-contract }
+
+To start interacting with [ChatGPT](https://chat.openai.com/?model=gpt-4){target=_blank}, take the following steps:
+
+1. Press **New Chat** in the upper left hand corner
+2. Select the model you would like to use. Either model is suitable for this tutorial
+3. Enter your prompt and at the input box and press enter when ready
+
+![Prompt chatGPT](/images/tutorials/eth-api/chatgpt/chatgpt-3.png)
+
+For our first prompt, we'll ask ChatGPT to create an ERC-20 token, specifying the name of the token, the token symbol, and an initial supply. Your prompt doesn't need to match the one below - feel free to tailor it to suit your preferences.
+
+```
+I would like to create an ERC-20 token called "KevinToken" with the symbol "KEV" and an initial supply of 40000000.
```

Comment:
I don't know if it would make sense to add a line break here, but maybe? I would like to be able to read the sentence without having to scroll but also I know it's weird when copied because the line break is there - not sure, what do you think?

Reply:
Fixed! Added a new line without the new line character and it looks perfect when copy pasted